### PR TITLE
Increase AbstractIntegerAssert test coverage to 100%

### PR DESCRIPTION
**Pull request by sarajuhosova**
#### Check List:
* Issue : NA
* Unit tests : YES
* Javadoc with a code example (on API only) : NA
* PR meets the [contributing guidelines](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md)

Following the [contributing guidelines](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md) will make it easier for us to review and accept your PR.

### CODE REVIEW

File: src/test/java/org/assertj/core/api/integer_/IntegerAssert_isNotCloseTo_int_Test.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.assertj.core.api.IntegerAssertBaseTest;
+import org.assertj.core.data.Offset;
+
+import static org.assertj.core.data.Offset.offset;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Tests for <code>{@link IntegerAssert#isNotCloseTo(Integer, Offset)}</code>.
+ *
+ * @author Chris Arnott
```
Comment (scordio) [inline src/test/java/org/assertj/core/api/integer_/IntegerAssert_isNotCloseTo_int_Test.java:25]: Would you like to update the `@author` with your name? Same for the other tests.

**Review by scordio — COMMENTED** [review]
Thanks, @sarajuhosova! Just one comment.

File: src/test/java/org/assertj/core/api/integer_/IntegerAssert_isNotCloseTo_int_Test.java (hunk 1 above)
Comment (sarajuhosova) [reply src/test/java/org/assertj/core/api/integer_/IntegerAssert_isNotCloseTo_int_Test.java]: Oh, thanks for the pointer, it's been updated :relaxed:

**Comment by joel-costigliola** [comment]
Integrated thanks @sarajuhosova!

**Pull request merged**